netmanager/src/assets/css/dropdown.css

```diff
@@ -1,129 +1,69 @@
-/*a {*/
-/*  text-decoration: none;*/
-/*  color: #000000;*/
-/*}*/
-
-/*a:hover {*/
-/*  color: #222222*/
-/*}*/
-
-/* Dropdown */
-
-.dropdown {
-  display: inline-block;
+.dropdown-wrapper {
+  /* display: flex; */
+  /* align-items: center; */
```

Comment:
Thanks @danielmarv , basing on the deploy preview link, all looks good to me now.

I request you to consult with your supervisor about the CSS changes in this file. Just want to make sure that the rest of the system remains intact.

Once you have alignment in form of an approval from your supervisor, the PR will be merged.

Thanks!

Reply:
Awesome work @danielmarv 
Some small feedback: Could we adjust the dropdown button's height to match that of the refresh button? Also, the font size of the items within the dropdown seems pretty small on my end, may be setting them to 18px..

![image](https://github.com/airqo-platform/AirQo-frontend/assets/46527380/96fda310-d25f-40eb-aeca-8a3ee7f73e34)